datafusion/sql/tests/sql_integration.rs

```diff
@@ -233,6 +233,25 @@ fn plan_create_table_with_pk() {
     let sql = "create table person (id int, name string, primary key(id))";
     let plan = r#"
 CreateMemoryTable: Bare { table: "person" } constraints=[PrimaryKey([0])]
+  EmptyRelation
+    "#
+    .trim();
+    quick_test(sql, plan);
+
+    let sql = "create table person (id int primary key, name string)";
```

Comment:
What do you think about adding unique constraints as well

```sql
create table person (id int unique, name string)
```

Reply:
I add a test case about unique constraints.